connectors/es/client.py

```diff
@@ -40,7 +40,12 @@ class License(Enum):
     UNSET = None
 
 
+USER_AGENT_BASE = f"elastic-connectors-{__version__}"
+
+
 class ESClient:
```

Comment:
`ESManagementClient` shares the same user agent with `ESClient`, and there's no way to differentiate them.

`ESClient` is used here:

https://github.com/elastic/connectors/blob/34ff685d7d83d4ab251ff7bb275fcbfb6c7ef479/connectors/es/index.py#L20

Reply:
> ESManagementClient shares the same user agent with ESClient, and there's no way to differentiate them.

I'm not quite sure, if I understand it correctly. Could you elaborate what's the potential problem?

Reply:
Yeah, we don't need to distinguish between `ESManagementClient` and `ESClient`. We just want to know, whether the `service` or the `cli` made a call.